Add non-graphical vimdiff to and remove tools irrelevant to a Debian system
(especially filemerge, see Issue1459) from mergetools.hgrc

diff -r 11a4eb81fb4f contrib/mergetools.hgrc
--- a/contrib/mergetools.hgrc
+++ b/contrib/mergetools.hgrc
@@ -12,8 +12,8 @@
 gvimdiff.regname=path
 gvimdiff.priority=-9
 
-merge.checkconflicts=True
-merge.priority=-10
+vimdiff.args=$local $other $base
+vimdiff.priority=-10
 
 gpyfm.gui=True
 
@@ -38,14 +38,6 @@
 p4merge.gui=True
 p4merge.priority=-8
 
-tortoisemerge.args=/base: $output /mine:$local /theirs:$other /merged:$output
-tortoisemerge.regkey=Software\TortoiseSVN
-tortoisemerge.gui=True
-
 ecmerge.args=$base $local $other --mode=merge3 --title0=base --title1=local --title2=other --to=$output
 ecmerge.regkey=Software\Elli\xc3\xa9 Computing\Merge
 ecmerge.gui=True
-
-filemerge.executable=/Developer/Applications/Utilities/FileMerge.app/Contents/MacOS/FileMerge
-filemerge.args=-left $other -right $local -ancestor $base -merge $output
-filemerge.gui=True
